src/chunkserver/network_worker_thread.cc

```diff
@@ -1633,35 +1585,17 @@ void NetworkWorkerThread::operator()() {
 void NetworkWorkerThread::terminate() {
 	TRACETHIS();
 	job_pool_delete(bgJobPool_);
+
```

Comment:
Should be considered to keep these extra spaces in the changes?

Reply:
Hi Roly, if you mean the empty lines, they are intentional, to visually separate the code blocks. IMO it is easier to read this way.